governance/votes/Executive vote - February 07, 2024.md

```diff
@@ -0,0 +1,125 @@
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Auction Parameter Updates
+
+* **Authorization**: [Governance Poll](https://vote.makerdao.com/polling/QmWLyYW7)
+* **Proposal**: [Forum post "[Stability Scope Auction Parameters Changes #1] Liquidation Throughput Limit"](https://forum.makerdao.com/t/stability-scope-auction-parameters-changes-1-liquidation-throughput-limit/23508)
+
+If this executive proposal passes, the following auction parameter changes will be changed:
+
+#### [Local Liquidation Limit (ilk.hole)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-5-8-local-liquidation-limit-hole-) Changes
+
+* Increase the WSTETH-A Local Liquidation Limit (ilk.hole) by 15 million DAI from 15 million DAI to **30 million DAI**.
+* Increase the WSTETH-B Local Liquidation Limit (ilk.hole) by 10 million DAI from 10 million DAI to **20 million DAI**.
+* Decrease the WBTC-A Local Liquidation Limit (ilk.hole) by 20 million DAI from 30 million DAI to **10 million DAI**.
+* Decrease the WBTC-B Local Liquidation Limit (ilk.hole) by 5 million DAI from 10 million DAI to **5 million DAI**.
+* Decrease the WBTC-C Local Liquidation Limit (ilk.hole) by 10 million DAI from 20 million DAI to **10 million DAI**.
+
+#### [Global Liquidation Limit (Hole)](https://manual.makerdao.com/parameter-index/core/param-global-liquidation-limit) Change
+
+* Increase the Global Liquidation Limit (Hole) by 50 million DAI from 100 million DAI to **150 million DAI**.
+
+### GUSD Input Conduit Management
+
+* **Authorization**: [Ecosystem Appoval](https://forum.makerdao.com/t/executive-spell-gusd-input-conduit-management/23597/2)
+* **Proposal**: [Forum post "Executive Spell GUSD Input Conduit Management"](https://forum.makerdao.com/t/executive-spell-gusd-input-conduit-management/23597)
+
+If this executive proposal passes, the following actions will be taken to `push` the remaining GUSD out of the PSM_GUSD_INPUT_CONDUIT_JAR, exchange it for Dai, and add it to the Surplus Buffer:
+
+1. Raise Debt Ceiling of PSM_GUSD_A to 567,660 DAI - accounts for the outstanding debt of 0.91 DAI of the vault.
+2. Call `push(uint256 amt)` on the the [PSM_GUSD_A_INPUT_CONDUIT_JAR](https://etherscan.io/address/0x6934218d8b3e9ffcabee8cd80f4c1c4167afa638) contract with an `amt` of 597,659, depositing the contents of the contract to the PSM and receiving an identical amount of Dai into the [PSM_GUSD_A_JAR](https://etherscan.io/address/0xf2e7a5b83525c3017383deed19bb05fe34a62c27) contract.
+3. Call `void` on the [PSM_GUSD_A_JAR](https://etherscan.io/address/0xf2e7a5b83525c3017383deed19bb05fe34a62c27) contract to deposit the Dai to the Surplus Buffer.
+4. Decrease Debt Ceiling of PSM_GUSD_A back to 0 DAI.
+
+This will have the following net results:
+
+1. Debt Ceiling remains at 0 after the transaction has executed.
+2. Dai minted from PSM_GUSD_A increases to 567,659.91 DAI.
```

Comment:
Maybe phrasing it as "temporary increases" would better communicate this action.